<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C74.0 CPU Debugger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="memoryBaseAddress">
               <property name="suffix">
                <string/>
               </property>
               <property name="prefix">
                <string>0x</string>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
               <property name="singleStep">
                <number>4</number>
               </property>
               <property name="displayIntegerBase">
                <number>16</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="memoryCount">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>64</number>
               </property>
               <property name="value">
                <number>16</number>
               </property>
               <property name="displayIntegerBase">
                <number>16</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="refreshMemory">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="res.qrc">
                 <normaloff>:/icons/iconfinder_sync_126579.png</normaloff>:/icons/iconfinder_sync_126579.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="addressToWrite">
               <property name="prefix">
                <string>0x</string>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
               <property name="singleStep">
                <number>4</number>
               </property>
               <property name="displayIntegerBase">
                <number>16</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="valueToWrite">
               <property name="inputMask">
                <string>HHHHHHHH</string>
               </property>
               <property name="placeholderText">
                <string>xxxxxxxx</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="writeMemButton">
               <property name="text">
                <string>Write</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="StatusLedsPanel" name="statusLedsPanel" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRescan">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionHalt"/>
    <addaction name="actionContinue"/>
    <addaction name="actionStep"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh_Registers"/>
   </widget>
   <addaction name="menuRescan"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHalt"/>
   <addaction name="actionContinue"/>
   <addaction name="actionStep"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRescan_comports">
   <property name="text">
    <string>Rescan comports</string>
   </property>
  </action>
  <action name="actionHalt">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconfinder_stop_326597.png</normaloff>:/icons/iconfinder_stop_326597.png</iconset>
   </property>
   <property name="text">
    <string>Halt</string>
   </property>
  </action>
  <action name="actionContinue">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/iconfinder_play_126575.png</normaloff>:/icons/iconfinder_play_126575.png</iconset>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/icons8-step-into-48.png</normaloff>:/icons/icons8-step-into-48.png</iconset>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRefresh_Registers">
   <property name="text">
    <string>Refresh Registers</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StatusLedsPanel</class>
   <extends>QWidget</extends>
   <header>statusledspanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
